app/src/main/res/layout/activity_checkout.xml

```diff
@@ -67,14 +67,14 @@
             android:layout_marginTop="@dimen/margin_xlarge">
 
             <include
-                android:id="@+id/googlepay_button"
+                android:id="@+id/mGooglePayButton"
```

Comment:
why?

Reply:
I use this as variables in Kotlin, so ids should follow field naming convention. Removed the m.